docker/scripts/entrypoint.sh

```diff
@@ -10,16 +10,14 @@
 # Environment variables in the form `ZKV_CONF_*` are translated to command line arguments based on these rules:
 #
 # 1. `ZKV_CONF_` prefix is removed
-# 2. if both a trailing underscore (`_`) and number are present, they are removed
```

Comment:
Why are you removing this behavior? we introduced it for every cases where we need to map array flags: flags that can be used more than once like `--bootnodes`.

Reply:
This is not being removed, rather being improved. With new implementation you no longer need to pass ONE value per VARIABLE if it supports list. Now you can do the following:

OLD
```
ZKV_CONF_LISTEN_ADDR="/ip4/0.0.0.0/tcp/${NODE_NET_P2P_PORT}"
ZKV_CONF_LISTEN_ADDR_2="/ip4/0.0.0.0/tcp/${NODE_NET_P2P_PORT_WS}/ws"
```

NEW
```
ZKV_CONF_LISTEN_ADDR="/ip4/0.0.0.0/tcp/${NODE_NET_P2P_PORT},/ip4/0.0.0.0/tcp/${NODE_NET_P2P_PORT_WS}/ws"
```

Reply:
Ok, I got it. We chose the other solution because it also possible to use it when the arguments can contain also `,`. But it's ok for now.... if we'll meet some issue we'll change it later.